cmatrix.c

```diff
@@ -81,6 +81,8 @@ int xwindow = 0;
 int lock = 0;
 cmatrix **matrix = (cmatrix **) NULL;
 int *length = NULL;  /* Length of cols in each line */
+int space = 50;      /* vertical density parameter */
```

Comment:
Can we make them constants? (I am not sure if we modify them later, can you check?)